atd-vze/src/views/Crashes/ChargesDetailsCard.js

```diff
@@ -47,6 +49,8 @@ const ChargesDetailsCard = ({ chargesData, isExpanded, toggleAccordion }) => {
                 (charge, i) =>
                   charge.charge !== "NO CHARGES" && (
                     <tr key={`charges-${i}`}>
+                      <td>{charge.unit_nbr}</td>
+                      <td>{charge.prsn_nbr}</td>
                       <td>{charge.charge}</td>
                       <td>{charge.charge_cat_id}</td>
```

Comment:
noticed that the `charge_cat_id` is null for every entry in the csvs you provided @johnclary ?

Reply:
ah yes—that is just CRIS being CRIS. it's always blank. it can be removed from the VZE ❌

Reply:
i see that we did get values for that in our old table, so is that just a field theyve since dropped?

Reply:
according to the CRIS extract specification it was deprecated in 2010:
<img width="323" alt="Screenshot 2024-05-29 at 12 48 20 PM" src="https://github.com/cityofaustin/atd-vz-data/assets/14793120/b31e2f88-438c-4c15-a040-ce7674ad5513">

so let's go ahead and remove it from the VZE 👍

Reply:
Cool added Citation Number as a field to the table